addons/sale/tests/test_reinvoice.py

```diff
@@ -176,8 +171,6 @@ def test_sales_price(self):
             'price_unit': self.company_data['product_order_sales_price'].list_price,
             'order_id': self.sale_order.id,
```

Comment:
you could reduce tests by removing all fields that are computed in every definition of SO line; price_unit, product_uom, name.

Reply:
Done at https://github.com/odoo/odoo/pull/79093/commits/2c73e6afffaa762dfc02cbe060c3f722f7639e96